src/main/java/frc/robot/commands/ShootingPosCommands/HumanPosition.java

```diff
@@ -1,27 +1,48 @@
 package frc.robot.commands.ShootingPosCommands;
 
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ConveyorSubsystem;
 import frc.robot.subsystems.ElevatorSubsystem;
+import frc.robot.subsystems.FlyWheelSubsystem;
 import frc.robot.subsystems.PivotSubsystem;
 
 public class HumanPosition extends Command {
-    PivotSubsystem pivot;
-    ElevatorSubsystem elevator;
+    private final PivotSubsystem pivot;
+    private final ElevatorSubsystem elevator;
+    private final FlyWheelSubsystem flywheel;
+    private final ConveyorSubsystem conveyor;
 
-    public HumanPosition(PivotSubsystem pivot, ElevatorSubsystem elevator){
+    public HumanPosition(PivotSubsystem pivot, ElevatorSubsystem elevator, FlyWheelSubsystem flywheel, ConveyorSubsystem conveyor){
         this.pivot = pivot;
         this.elevator = elevator;
+        this.flywheel = flywheel;
+        this.conveyor = conveyor;
     }
 
     @Override 
     public void initialize(){
         pivot.pivotHumanPosition();
         elevator.humanPosition();
+        flywheel.sourceNomNom();
+        conveyor.conveyFromSource();
+    }
+
+    @Override
+    public void execute(){
+        if(flywheel.isNoteSeen()){
+            conveyor.conveyFromSource();
+        }
     }
 
     @Override
     public boolean isFinished(){
-        return true;
+        return !flywheel.isNoteSeen();
```

Comment:
Will this cause the command to end instantly? I believe the linebreak sensor won't initially register the note as seen.

We probably need some kind of logic that checks that the note is seen first, then ends it once it isn't seen.

Reply:
Got it, sounds good. I was thinking of using an if statement because the LB will be trigged two times. So like
 if (LB) { is seen once check 
             if (LB) { then return is note seen or run conveyLBCommand }
            }

I feel like that can be a more simplified statement though, I just don't know if I'm on the right path or if there is a better way.

Reply:
That is the essence of the logic, but you'll need to translate that into code that will work with `execute` and `isFinished`. You probably need an if statement in `execute` to save if the note is seen the first time, and then only finish the command based on that.